src/components/browsePokemons/browsePokemons.css

```diff
@@ -0,0 +1,64 @@
+.container {
+  background-color: #1c1c1c !important;
+  min-height: 100vh !important;
+  padding: 3rem !important;
+}
+
+.grid-container {
+  display: flex !important;
+  flex-wrap: wrap !important;
+  gap: 2rem !important;
+}
+
+.card {

```

Comment:
So this entire file doesn't seem to be right...
The usage of `!important` is too much. you cannot do it like this because later you wont be able to change anything in your code since this !important will override it

Please remove it + check the documentation of `"@mui/material"` they probably have their own way of passing classes